lib/rules/prefer-ember-test-helpers.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { getImportIdentifier } = require('../utils/import');
+
+let hasImportedBlur = undefined;
+let hasImportedFind = undefined;
+let hasImportedFocus = undefined;
+
+//-------------------------------------------------------------------------------------
+// Rule Definition
+//-------------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'enforce import of @ember/test-helpers method to ensure native method on `window` is not incorrectly used',
+      category: 'Testing',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/prefer-ember-test-helpers.md',
+    },
+    schema: [],
+  },
+
+  create: context => {
+    const showErrorMessage = function(node, methodName) {
+      let errorMessage = 'Import the correct method from Ember test-helpers';
+
+      switch (methodName) {
```

Comment:
No need for a `switch` statement on this, just use:

```js
function makeErrorMessage(methodName) {
  return `Import the \`${methodName}\` method from @ember/test-helpers`
}
```

Reply:
Ooo so I got an eslint error when I tried that method first, which I misinterpreted as that I can't use template literals at all. https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/prefer-placeholders.md

Fixed!

Reply:
![image](https://user-images.githubusercontent.com/2523340/79028816-79d2b280-7b46-11ea-9673-89d2124ab772.png)